src/model/utils/errorResponse.ts

```diff
@@ -1,3 +1,11 @@
+import { MessageField } from "./messageField";
+
 export class ErrorResponse {
-    constructor(public error?: string, public message?: string){}
+    public error: string;
```

Comment:
Why would you want to remove constructors?

Reply:
This thing make the Fre8back didnt compile before. Dont know why it work now. Fixed . Changed it back